commit 8bd0f6ecd882a3119acb5fd7a760122c8931db94
Author: wanghong <w.wanghong@huawei.com>
Date:   Sat Oct 25 10:26:55 2014 +0800

    default revoke driver should be the non-deprecated driver
    
    Have checked other config options that default is not the deprecated
    option, others are OK.
    
    Change-Id: Ie6e7351114d79eb25b392faaa0e6376c24c45fa0
    Closes-Bug: #1384775

--- keystone-2014.2.2/etc/keystone.conf.sample.~1~	2014-12-04 21:03:56.000000000 -0800
+++ keystone-2014.2.2/etc/keystone.conf.sample	2015-01-25 17:25:46.677132074 -0800
@@ -1332,7 +1332,7 @@
 
 # An implementation of the backend for persisting revocation
 # events. (string value)
-#driver=keystone.contrib.revoke.backends.kvs.Revoke
+#driver=keystone.contrib.revoke.backends.sql.Revoke
 
 # This value (calculated in seconds) is added to token
 # expiration before a revocation event may be removed from the
--- keystone-2014.2.2/keystone/common/config.py.~1~	2014-12-04 21:03:56.000000000 -0800
+++ keystone-2014.2.2/keystone/common/config.py	2015-01-25 17:25:46.677474875 -0800
@@ -275,7 +275,7 @@ FILE_OPTIONS = {
     ],
     'revoke': [
         cfg.StrOpt('driver',
-                   default='keystone.contrib.revoke.backends.kvs.Revoke',
+                   default='keystone.contrib.revoke.backends.sql.Revoke',
                    help='An implementation of the backend for persisting '
                         'revocation events.'),
         cfg.IntOpt('expiration_buffer', default=1800,
